substrate/frame/vesting/Cargo.toml

```diff
@@ -26,6 +26,8 @@ sp-std = { path = "../../primitives/std", default-features = false}
 
 [dev-dependencies]
 pallet-balances = { path = "../balances" }
+pallet-root-testing = { path = "../root-testing" }
+pallet-collective = { path = "../collective" }
```

Comment:
These pallets are not used in tests, so all of the changes in this file are unnecessary.

Reply:
Yeah this is a hang over from earlier when I first started on the PR and was poking around a bit. Will remove them.